src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -223,14 +223,13 @@ export class CalcitePickListItem {
       ) : null;
 
     return (
-      <Host selected={this.isSelected}>
+      <Host selected={this.isSelected} role="checkbox" aria-checked={this.isSelected}>
```

Comment:
axe didn't like these attributes on the label tag.

Reply:
This is ok, but maybe it should go on the icon? The icon is essentially the replacement for the checkbox.

Reply:
We could do that, but then what do we do for valueListItems?

Reply:
I'm not sure, but I think the role should be tied to the icon node because the secondary action shouldn't be nested within the checkbox role.

Reply:
As discussed, opening a new issue to investigate further. Moving forward with this in the meantime.